packages/components/pie-checkbox/test/component/pie-checkbox.spec.ts

```diff
@@ -86,6 +86,55 @@ test.describe('PieCheckbox - Component tests', () => {
             });
         });
 
+        test.describe('checked', () => {
+            test('should default to false', async ({ mount, page }) => {
+                // Arrange
+                const component = await mount(PieCheckbox, {});
+
+                // Act
+                const checkbox = component.locator(componentSelector);
+                const isChecked = await page.evaluate(() => document.querySelector('pie-checkbox')?.checkbox?.checked);
+
+                // Assert
+                expect(await checkbox.isChecked()).toBe(false);
+                expect(isChecked).toBe(false);
+            });
+
+            test('should be unchecked if the checked prop is false', async ({ mount, page }) => {
+                // Arrange
+                const component = await mount(PieCheckbox, {
+                    props: {
+                        checked: false,
+                    } as CheckboxProps,
+                });
+
+                // Act
+                const checkbox = component.locator(componentSelector);
+                const isChecked = await page.evaluate(() => document.querySelector('pie-checkbox')?.checkbox?.checked);
+
+                // Assert
+                expect(await checkbox.isChecked()).toBe(false);
+                expect(isChecked).toBe(false);
+            });
+
+            test('should be checked if the checked prop is true', async ({ mount, page }) => {
+                // Arrange
+                const component = await mount(PieCheckbox, {
+                    props: {
+                        checked: true,
+                    } as CheckboxProps,
+                });
+
+                // Act
+                const checkbox = component.locator(componentSelector);
+                const isChecked = await page.evaluate(() => document.querySelector('pie-checkbox')?.checkbox?.checked);
+
+                // Assert
+                expect(await checkbox.isChecked()).toBe(true);
```

Comment:
Do we need to verify that the prop works as intended here as part of this & the above test? I think we probably don't need to IMO because the next line would fail if the prop wasn't being applied correctly to the component.